src/Test/Hspec/Hedgehog.hs

```diff
@@ -84,10 +84,12 @@ module Test.Hspec.Hedgehog
     ( -- * The Main Function
       hedgehog
       -- * Hspec re-exports
+      --
+      -- | 'Test.Hspec.QuickCheck.modifyMaxSize' isn't re-exported, since
+      -- hedgehog has nothing that corresponds to it.
```

Comment:
I am 50/50 on this one.  Is this useful information for someone who is going to use the API? Or is this just the justification for us to remove the re-export? If it's the later, then that piece of documentation can be removed.

@ChickenProp I defer to your judgement.

Reply:
I think useful, yeah. I've used this re-export in the past, not realizing it was useless. If it had been removed without saying why in the docs, I would have either spent a while trying to figure out why it was missing, or just taken the function from its original location.